src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -595,6 +595,9 @@ private bool ShouldTryToReconnect(Exception ex)
         {
             switch (ex)
             {
+                case InvalidOperationException _:
```

Comment:
why?

Reply:
When can we get this?

Reply:
This is what is thrown when an unexpected error is sent.
https://github.com/ravendb/ravendb/blob/a73056b86822a71a6401dd5475ea904cdabed732/src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs#L455

But probably need to set `MaxErroneousPeriod` instead.